samples/snippets/src/main/java/com/example/spanner/SpannerSample.java

```diff
@@ -488,44 +498,45 @@ static void update(DatabaseClient dbClient) {
   static void writeWithTransaction(DatabaseClient dbClient) {
     dbClient
         .readWriteTransaction()
-        .run(transaction -> {
```

Comment:
Here and elsewhere in this file: There seem to be a number of unrelated formatting changes. I think those should be removed from this PR, and if there is something wrong with the formatting here, we should fix that in a separate PR.